generation/new_client/README.md

```diff
@@ -124,55 +124,73 @@ $ sudo apt-get install gh
 $ gh auth login
 ```
 
+## Double check that the library does not already exist within `google-cloud-java`!
+
+Some languages require a new request when a new version of a service is created, but Java manages all versions of the service as a single package, so the client library will automatically be updated to include new versions.
+
 ## Run client generation script
 
 You will run new-client.py script with the following parameters.
-Collect them from the ticket before running the command.
+These parameters will be available in the Cloud Drop link (a YAML file) included in the buganizer request.
+The example in this README will AlloyDB's [Cloud Drop](https://github.com/googleapis/googleapis/blob/master/google/cloud/alloydb/v1/alloydb_v1.yaml) file as an example.
 
 ### API short name
 
 For convenience of the subsequent commands, define a variable for API short name.
-Get the value from the DevRel Services page (Example: `apikeys`):
+The corresponding value in the Cloud Drop page is `api_short_name`.
+
+Example: `alloydb`
 
 ### Proto path
 
-The script takes "proto path" parameter. This is path from google3's root to the
+The script takes "proto path" parameter. This is path from `googleapis/googleapis/blob/master`' root to the
 directory that contains versions (e.g., "v1" or "v2"). For example, if we
 have 
 
-> Link to protos: http://google3/google/api/apikeys/v2/apikeys.proto
+> Link to protos: https://github.com/googleapis/googleapis/blob/master/google/cloud/alloydb/v1/service.proto.
```

Comment:
This does not match the description in b/271166569. Where did this come?

Reply:
You would need an instruction to convert the "Link to protos: " value in the ticket to the proto path.